stock_picking_partner_note/models/stock_picking.py

```diff
@@ -8,14 +8,11 @@
 class StockPicking(models.Model):
     _inherit = "stock.picking"
 
-    note = fields.Html(compute="_compute_note", store=True)
+    note = fields.Text(compute="_compute_note", store=True)
 
-    # TODO: ideally, we should add more detailed dependencies here;
-    # however, there's a CacheMiss error that occurs when adding
-    # "picking_type_id.partner_note_type_ids" and "partner_id.stock_picking_note_ids"
-    # so we're leaving it like this for now
-    # (the partner should be enough to trigger the compute).
-    @api.depends("partner_id")
+    @api.depends(
+        "partner_id.stock_picking_note_ids", "picking_type_id.partner_note_type_ids"
```

Comment:
That will trigger a change in all picking notes if a single note is changed in the partner. We can't do that